sdk/function_id.go

```diff
@@ -0,0 +1,89 @@
+package sdk
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/pkg/errors"
+
+	"github.com/capeprivacy/cli/attest"
+	"github.com/capeprivacy/cli/crypto"
+	"github.com/capeprivacy/cli/entities"
+	"github.com/capeprivacy/cli/pcrs"
+)
+
+type FunctionIDRequest struct {
+	URL          string
+	FunctionName string
+	FunctionAuth entities.FunctionAuth
```

Comment:
`FunctionAuth`, `PcrSlice`, and `Insecure` are mostly all specific to the enclave, and this request does not use the enclave, so they could be removed. For the function ID, you are just going to be making a regular HTTP request

Also, a bit of a meta sidebar about the SDK, but I also think that a user of this SDK shouldn't have to specify the URL, and we should use prod as the default (cc @justin1121 and @jlapierre on this one, since they are the code owners)

Reply:
Agree with the bit about not having to specify the URL for users of the go SDK 👍

Reply:
default-to-prod sounds good to me